lizard_blockbox/migrations/0002_fixtures.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-05-01 13:41
+from __future__ import unicode_literals
+
+import os
+
+from django.db import migrations
+from django.core import serializers
+
+fixture_path = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '../fixtures'))
+
+
+def load_map_fixtures(apps, schema_editor):
```

Comment:
Dit is prima, maar ook hier wil ik je nog even op deze stackoverflow post willen wijzen: https://stackoverflow.com/questions/25960850/loading-initial-data-with-django-1-7-and-data-migrations/25981899#25981899